solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -67,7 +67,7 @@ describe("RandomBeacon - Relay", () => {
   let signersAddresses: Address[]
```

Comment:
It seems that we use `member`, `operator` & `signer` across the code to describe the same thing and it might be confusing.  I think it's harder to "merge" `member`&`operator` since ex. sortition pool uses `operator` and in groups lib we use `member`. But can we at least try to replace `signer` with `member`?

Reply:
Yeah, I don't like it either but there is no ideal solution here. Even `member` instead `signer` is not justified in all contexts because an operator becomes a member after group creation. But, your proposition sounds good for this use case and I changed accordingly: https://github.com/keep-network/keep-core/pull/2671/commits/d70c7ef883c278ed71d22275b8ec0b2e27bdf845